wkcuber/README.md

```diff
@@ -1,155 +1,49 @@
 
+### Python 3 with pip from PyPi
 
-## Usage
+- `wkcuber` requires at least Python 3.10
 
 ```bash
-# Convert arbitrary, supported input files into wkw datasets. This sets reasonable defaults, but see other commands for customization.
-python -m wkcuber \
-  --scale 11.24,11.24,25 \
-  data/source data/target
-
-# Convert image stacks into wkw datasets
-python -m wkcuber.convert_image_stack_to_wkw \
-  --layer_name color \
-  --scale 11.24,11.24,25 \
-  --name great_dataset \
-  data/source/color data/target
-
-# Convert image files to wkw cubes
-python -m wkcuber.cubing --layer_name color data/source/color data/target
-python -m wkcuber.cubing --layer_name segmentation data/source/segmentation data/target
-
-# Convert tiled image files to wkw cubes
-python -m wkcuber.tile_cubing --layer_name color data/source data/target
-
-# Convert Knossos cubes to wkw cubes
-python -m wkcuber.convert_knossos --layer_name color data/source/mag1 data/target
-
-# Convert NIFTI file to wkw file
-python -m wkcuber.convert_nifti --layer_name color --scale 10,10,30 data/source/nifti_file data/target
-
-# Convert folder with NIFTI files to wkw files
-python -m wkcuber.convert_nifti --color_file one_nifti_file --segmentation_file --scale 10,10,30 another_nifti data/source/ data/target
-
-# Convert RAW file to wkw file
-python -m wkcuber.convert_raw --layer_name color --scale 10,10,30 --input_dtype uint8 --shape 2048,2048,1024 data/source/raw_file.raw data/target
-
-# Create downsampled magnifications
-python -m wkcuber.downsampling --layer_name color data/target
-python -m wkcuber.downsampling --layer_name segmentation --interpolation_mode mode data/target
-
-# Compress data in-place (mostly useful for segmentation)
-python -m wkcuber.compress --layer_name segmentation data/target
-
-# Compress data copy (mostly useful for segmentation)
-python -m wkcuber.compress --layer_name segmentation data/target data/target_compress
-
-# Create metadata
-python -m wkcuber.metadata --name great_dataset --scale 11.24,11.24,25 data/target
-
-# Refresh metadata so that new layers and/or magnifications are picked up
-python -m wkcuber.metadata --refresh data/target
-
-# Recubing an existing dataset
-python -m wkcuber.recubing --layer_name color --dtype uint8 /data/source/wkw /data/target
+pip install wkcuber
 
-# Check two datasets for equality
-python -m wkcuber.check_equality /data/source /data/target
+# to install auto completion as well use:
+wkcuber --install-completion
 ```
 
-### Parallelization
 
-Most tasks can be configured to be executed in a parallelized manner. Via `--distribution_strategy` you can pass `multiprocessing`, `slurm` or `kubernetes`. The first can be further configured with `--jobs` and the latter via `--job_resources='{"mem": "10M"}'`. Use `--help` to get more information.
-
-### Zarr support
```

Comment:
Can you check that the Zarr support still works? I think there needs to be some supporting code in wkcuber to translate the env vars into UPaths.